### DISCUSSION TITLE
Support for Python for Qt (PySide2)

### DISCUSSION THREAD

**Discussion by Chaz6** [mu-editor/mu#1846] [Ideas & New Features]
mu currently requires PyQt. It would be nice to be able to use Python for Qt (PySide2) instead.

**Comment by carlosperate** [comment]
Hi @Chaz6,
My first impression is that even if it's mostly just changing a few API calls here and there, it could still end up being a significant change without any noticeable impact to the end user.
To better understand the suggestion I was wondering if you could expand a bit more on why would the change be beneficial?

**Comment by ntoll** [comment]
:wave: I have wondered about this too, however, the PyQScintilla widget we use for the main editor window is, I believe, a PyQt "thing" and wouldn't work with PySide2. (See: https://www.riverbankcomputing.com/software/qscintilla/download)
